src/app/qgsprojectlistitemdelegate.cpp

```diff
@@ -168,7 +180,7 @@ QPixmap QgsProjectPreviewImage::pixmap() const
   QPainter previewPainter( &previewImage );
   previewPainter.setRenderHint( QPainter::Antialiasing, true );
   previewPainter.setPen( Qt::NoPen );
-  previewPainter.setBrush( Qt::black );
+  previewPainter.setBrush( QBrush( QColor( 0, 0, 0 ), Qt::SolidPattern ) );
```

Comment:
what impact does this change have?

Reply:
@nyalldawson , that fixes nothing indeed, reverted, leftover from some debugging early on.